dotcom-rendering/src/web/components/Doughnut.tsx

```diff
@@ -40,33 +39,33 @@ const withoutZeroSections = (sections: Section[]) =>
 export const Doughnut = ({
 	sections,
 	percentCutout = 35,
-	width = 300,
-	height = 300,
+	size = 300,
 }: Props) => {
-	if (withoutZeroSections(sections).length === 1) {
-		// The Doughnut component requires at least 2 sections
-		// TODO: Support showing 100% for a single section
-		return null;
-	}
-
 	// TODO: Support displaying 0% for sections where value is zero
 	// We handle these at the moment by filtering them out using withoutZeroSections()
 
-	const radius = Math.min(height / 2, width / 2);
-	const cutoutRadius = radius * (percentCutout / 100);
+	const outerRadius = size / 2;
+	const innerRadius = outerRadius * (percentCutout / 100);
+	const radius = (innerRadius + outerRadius) / 2;
 
 	const totalValue = sections
 		.map((section) => section.value)
 		.reduce((runningTotal, currentValue) => runningTotal + currentValue);
 
-	const halfPI = Math.PI / 2;
-	const doublePI = Math.PI * 2;
+	/** τ = 2π https://en.wikipedia.org/wiki/Turn_(angle)#Tau_proposals */
+	const tau = Math.PI * 2;
 
 	const center = {
-		x: width / 2,
-		y: height / 2,
+		x: size / 2,
```

Comment:
Worth refactoring this into a single variable since x and y will be identical now?